src/test/suite/z_extension.test.ts

```diff
@@ -26,80 +26,60 @@ import { ACTIVATION_TIMEOUT } from './Utils';
 suite("ensure camelk extension exists and is accessible", function() {
 	const extensionId = 'redhat.vscode-camelk';
 
-	test('vscode-camelk extension should be present', function(done) {
+	test('vscode-camelk extension should be present', function() {
 		assert.ok(vscode.extensions.getExtension(extensionId));
-		done();
 	});
 
 	test('vscode-camelk extension should activate', async() => {
 		await utils.ensureExtensionActivated();
 	}).timeout(ACTIVATION_TIMEOUT + 1000);	
 
-	test('test optional namespace support', function(done) {
+	test('test no namespace provided to getBaseCmd doesn\'t contain --namespace parameter', function() {
 		let cmdStrNoNS : string = kamel.getBaseCmd(`fakepath`,`fakecommand`, undefined);
 		assert.equal(cmdStrNoNS.indexOf('--namespace'), -1);
-
+	});
+	
+	test('test namespace provided to getBaseCmd contain --namespace parameter', function() {
 		let cmdStrWithNS : string = kamel.getBaseCmd(`fakepath`,`fakecommand`, 'fakens');
-		assert.equal(cmdStrWithNS.indexOf('--namespace') > 0, true);
-
-		done();
+		assert.ok(cmdStrWithNS.includes('--namespace'));
 	});
 
 	test('test setting namespace to undefined', async function() {
-		// get NS from config settings
 		const namespace : string | undefined = config.getNamespaceconfig();
-
-		// reset to undefined
 		await config.addNamespaceToConfig(undefined);
 
-		// get NS from config settings
-		const resetNs : string | undefined = config.getNamespaceconfig();
-
-		// by default this should be undefined
-		assert.equal(resetNs, undefined);
-
-		// reset to old value
+		const namespaceConfigValueAfterReset : string | undefined = config.getNamespaceconfig();
+		assert.equal(namespaceConfigValueAfterReset, undefined, 'By default, the namespace config should be undefined');
+		
 		await config.addNamespaceToConfig(namespace);
 	});
 
 	test('test setting namespace to other value', async function() {
-		// get NS from config settings
 		const namespace : string | undefined = config.getNamespaceconfig();
-
-		// get NS from config settings
 		const testNs = 'testing';
-
-		// override namespace
 		await config.addNamespaceToConfig(testNs);
 
-		// re-retrieve namespace, should be test NS we specified
-		const resetNs : string | undefined = config.getNamespaceconfig();
+		const namespaceConfigValueAfterReset : string | undefined = config.getNamespaceconfig();
+		assert.equal(namespaceConfigValueAfterReset, testNs, 'The setted value should be available.');
```

Comment:
"setted" is not really a word -- perhaps change to "The value set should be available"